.changeset/twelve-rules-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+Added check to detect `jest`,`vitest` and `mocha` testing framework.
```

Comment:
```suggestion
---
'@itwin/itwinui-react': patch
---

iTwinUI will now also check for `vitest` and `mocha` (in addition to `jest`) before running any code that would cause problems in these test runners.
```